checkov/main.py

```diff
@@ -75,6 +76,8 @@
 logger = logging.getLogger(__name__)
 checkov_runners = [value for attr, value in CheckType.__dict__.items() if not attr.startswith("__")]
 
+RUN_SCA_PACKAGE_SCAN_V2 = os.getenv('RUN_SCA_PACKAGE_SCAN_V2', '').lower() == 'true'
```

Comment:
```suggestion
RUN_SCA_PACKAGE_SCAN_V2 = os.getenv('CHECKOV_RUN_SCA_PACKAGE_SCAN_V2', '').lower() == 'true'
```
we usually prefix internal env vars with `CHECKOV` which the user should only change, if he knows what he is doing 😄

Reply:
and move the const to here `checkov/common/util/consts.py` then import it from there, so you don't need to retrieve it in different areas